### PR TITLE
Added new actions for Volume and Volume-Action

### PR DESCRIPTION
**Pull request by xmudrii**
I gave it a try, I hope it's OK :+1: =)

cc @aybabtme

### CODE REVIEW

File: do/volumes.go [hunk 1]
```diff
@@ -7,12 +7,13 @@ type Volume struct {
 	*godo.Volume
 }
 
-// VolumesService is an interface for interacting with DigitalOcean's account api.
+// VolumesService is an interface for interacting with DigitalOcean's volume api.
```
Comment (aybabtme) [inline do/volumes.go]: nice catch

**Comment by nanzhong** [comment]
lgtm but will need a rebase and small change now that https://github.com/digitalocean/doctl/pull/171 is merged right?

**Comment by xmudrii** [comment]
@nanzhong is it good now? :)

File: args.go [hunk 2]
```diff
@@ -134,4 +134,6 @@ const (
 
 	// ArgResourceType is the resource type for snapshot.
 	ArgResourceType = "resource"
+	// ArgSnapshotDesc is the description for volume snapshot.
+	ArgSnapshotDesc = "snapshot-desc"
```
Comment (nanzhong) [inline args.go]: Sorry tiny nitpick I just noticed, could we move this up to under `ArgSnapshotName` to make it a little easier to find? https://github.com/digitalocean/doctl/pull/173/files#diff-07ee57a49d1c0b692bd79676cc31dd6fR54

I'll merge this PR after that.

Comment (xmudrii) [inline args.go]: Not a problem, thanks for noting. Moved both up. :+1:

**Pull request merged**